js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/Conditions.tsx

```diff
@@ -42,9 +45,11 @@ export const ConditionsWithPartitions = ({
   maxMaterializationsPerMinute,
   parentOutdatedWaitingOnAssetKeys,
 }: ConditionsWithPartitionsProps) => {
-  const buildRightElement = (partitionKeys: string[]) => {
+  const buildRightElement = (partitionKeys: string[], intent?: any) => {
```

Comment:
Avoid `any` where possible, since it ejects us out of type safety.

You can use `React.ComponentProps<typeof Tag>['intent']` here, where `Tag` is the `@dagster-io/ui` import.